src/software/ai/navigator/path_manager/obstacle_path_manager.cpp

```diff
@@ -0,0 +1,79 @@
+#include "software/ai/navigator/path_manager/obstacle_path_manager.h"
+
+/**
+ * This file contains the implementation of the obstacle path manager
+ * which uses obstacles to arbitrate between paths
+ * ret a path consisting of only the start and destination
+ * points.
```

Comment:
I think that this point we've somewhat unofficially come to the consensus of just leaving a class level jdoc in the header to avoid duplicate information. I don't feel super strongly about this. If you decide to keep it, the last two lines are somewhat confusion and seem out of place?

Reply:
removed